hasjob/models/jobpost.py

```diff
@@ -443,6 +444,10 @@ def uncache_viewcounts(self, key=None):
         else:
             redis_store.hdel(cache_key, key)
 
+    @staticmethod
+    def max_counts_key(postids):
+        return "hasjob/maxcounts/{hash}".format(hash=sha256(','.join(str(postid) for postid in postids)).hexdigest())
```

Comment:
You don't need this. The maxcount is global, for all of Hasjob, not just for the view. Save the key as `hasjob/maxcounts` and drop the postids. This static method is unnecessary.